api/turing/config/example.yaml

```diff
@@ -209,10 +209,13 @@ TuringUIConfig:
 # Openapi validation configuration
 OpenapiConfig:
   ValidationEnabled: true
-  SpecFile: "api/openapi.yaml"
+  # This file will be ignored by the OpenAPI validator if SwaggerUIConfig is filled.
+  SpecFile: api/openapi.yaml
+  SpecOverrideFile: api/override-sample.yaml
   SwaggerUIConfig:
     ServingDirectory: static/swagger-ui
     ServingPath: /api-docs
+    YAMLServingURL: /static/openapi.bundle.yaml
```

Comment:
Isn't it expected to be at the same level as `SwaggerUIConfig` and not nested under it?

Reply:
ya good catch.